<style type="text/css">
    #home,
    #program,
    #registration,
    #venue,
    #contact {
        scroll-margin-top: 11rem;
    }
    .home {
        animation: bgFadeIn 1s ease-in both;
    }
    @keyframes bgFadeIn {
        0% {
        background-color: rgb(252, 253, 255);
        }
        100% {
        background-color: rgba(252, 253, 255, 0);
        }
    }
    .btn-register,
    .btn-register:active,
    .btn-register:focus {
        color: #fcfdff;
        border-color: #fcfdff;
        padding: 1em;
        position: relative;
        overflow: hidden;
        z-index: 9;
        text-decoration: none;
    }
    .btn-register:hover {
        color: #fcfdff;
        background-color: rgba(222, 227, 234, 0);
        text-decoration: none;
    }
    .btn-register:before {
        content: "";
        background: #109e9b;
        width: 0px;
        height: 100%;
        position: absolute;
        left: 0;
        top: 0;
        z-index: -1;
        -webkit-transition: all 0.25s ease-in-out 0s;
        -moz-transition: all 0.25s ease-in-out 0s;
        -ms-transition: all 0.25s ease-in-out 0s;
        -o-transition: all 0.25s ease-in-out 0s;
        transition: all 0.25s ease-in-out 0s;
    }
    .btn-register:hover:before {
        width: 100%;
    }
    .border {
        border-color: #fcfdff !important;
        transition: border 0.4s ease-in-out;
    }
    .border:hover {
        border-color:#109e9b !important;
        transition: border 0.4s ease-in-out;
    }
    @media only screen and (max-width: 913px) {
        .description{
            margin-top: 1.5rem;
            min-height: 200px;
        }
    }
</style>

<!--Scrollspy-->
<div data-bs-spy="scroll" data-bs-target="#navbar" data-bs-offset="0" tabindex="0">

    <!--Welcome section-->
    <section class="home py-5">
        <div class="container" id="home">
            <p class="fs-2">Celebrating 15 Years of BioQuant</p>
            <p>Welcome to BioQuant, the Center for Quantitative Analysis of Molecular and Cellular Biosystems!</p>
            <p>Founded in 2007, our institute has served to advance biomedical research and education. Since then we have grown into a hub of the multidisciplinary research network. And this year we are celebrating BioQuant's 15th anniversary.</p>
            <p>To mark the occasion, we are hosting a scientific symposium and a jubilee party. We cordially invite all current and former members of BioQuant to join us on Tuesday, June 21, 2022.</p>
            <p>We are very pleased to welcome Prof. Hans-Georg Kräusslich and Prof. Jürgen Wolfrum, two of the founding directors of BioQuant, as well as Prof. Ana Garcia Sáez, Prof. Barbara Di Ventura, Prof. Dirk-Peter Herten and Prof. Lars Kaderali - all very successful former BioQuant research group leaders  - as speakers for the BioQuant Symposium.</p>
        </div>
    </section>

    <div class="full-width"><img class="img-fluid w-100" alt="Group of researchers" src="/sites/default/files/15years/fireworks.jpg"/></div>

    <!--Program section-->
    <section class="program text-white bg-black full-width py-5">
        <div class="container" id="program">
    
            <p class="fs-2">Program</p>
            <p class="fs-4">Scientific Symposium and Jubilee Party</p>
            <p class="fs-5 text-green">Tuesday, June 21, 2022</p>
            <p>All talks will take place in seminar rooms 41-44.</p>

            <div class="row program-item text-white bg-black py-3 px-0">
                <div class="col-md-2 pb-3">12.00 PM</div>
                <div class="col fs-5"">Registration @ Foyer</div>
            </div>
            <div class="row program-item text-white bg-black py-3 px-0">
                <div class="col-md-2 pb-3 pb-3">01.00 PM</div>
                <div class="col">
                    <p class="fs-5">Opening</p>
                    <p>by Hans-Georg Kräusslich</p>
                    <p>Center for Infectious Diseases, Heidelberg University Hospital</p>
                </div>
            </div>
            <div class="row program-item text-white bg-black py-3 px-0">
                <div class="col-md-2 pb-3">01.30 PM</div>
                <div class="col">
                    <p class="fs-5">Shedding new light on mitochondrial apoptosis</p>
                    <p>by Ana J. Garcia-Saéz</p>
                    <p>CECAD Research Center and Institute for Genetics, University of Cologne</p>
                    <p>
                        <button class="btn btn-abstract text-white border-white" type="button" data-bs-toggle="collapse" data-bs-target="#collapseAbstract1" aria-expanded="false" aria-controls="collapseAbstract1">
                            Read the abstract
                        </button>
                    </p>
                    <div class="collapse" id="collapseAbstract1">
                        <div class="card card-body text-white bg-black">
                            BAX and BAK are key apoptosis regulators that mediate the decisive step of mitochondrial outer membrane permeabilization. However, the mechanism by which they assemble the apoptotic pore remains obscure. I will discuss our work using a combination of single molecule and super resolution microscopy methods. We found that BAX and BAK present distinct oligomerization properties, and that BAK recruits and accelerates BAX assembly into oligomers that continue to grow during apoptosis. We visualized how BAX and BAK regulate each other as they co-assemble into the same apoptotic pores. Our results indicate that the relative availability of BAX and BAK molecules determines the growth rate of the apoptotic pore and the relative kinetics by which mitochondrial contents, most notably mtDNA, are released. This feature of BAX and BAK results in distinct activation kinetics of the cGAS/STING pathway with implications for mtDNA-mediated paracrine inflammatory signaling.
                        </div>
                    </div>
                </div>
            </div>
            <div class="row program-item text-white bg-black py-3 px-0">
                <div class="col-md-2 pb-3">02.00 PM</div>
                <div class="col">
                    <p class="fs-5">Mathematical modeling and the pandemic: From replication models to lockdown decisions</p>
                    <p>by Lars Kaderali</p>
                    <p>Institute for Bioinformatics, Greifswald University Hospital</p>
                    <p>
                        <button class="btn btn-abstract text-white border-white" type="button" data-bs-toggle="collapse" data-bs-target="#collapseAbstract2" aria-expanded="false" aria-controls="collapseAbstract2">
                            Read the abstract
                        </button>
                    </p>
                    <div class="collapse" id="collapseAbstract2">
                        <div class="card card-body text-white bg-black">
                            The past two years of the Covid pandemic have made mathematical modeling of viral infections broadly visible to the general public. Models have been used to predict future infection numbers as much as number of patients in intensive care and on ventilators, with results being presented in TV talk shows as well as in executive government meetings and parliament discussions. It is fair to say that mathematical models have had a significant impact on political decisions and on the course of the pandemic in the past 2 years. Interestingly, there is no strong scientific community on epidemiological modeling in Germany. Most models have been developed by groups that are rooted in different fields. I will talk about about the journey I have taken, from developing mathematical models of viral replication and immune response at the level of individual cells at BioQuant, their extension to patient and drug treatment data, to recent models that we have developed in the context of the Covid-19 pandemic.
                        </div>
                    </div>
                </div>
            </div>
            <div class="row program-item text-white bg-black py-3 px-0">
                <div class="col-md-2 pb-3">02.30 PM</div>
                <div class="col fs-5">Coffee break @ Foyer</div>
            </div>
            <div class="row program-item text-white bg-black py-3 px-0">
                <div class="col-md-2 pb-3">03.00 PM</div>
                <div class="col">
                    <p class="fs-5">A question of dynamics</p>
                    <p>by Barbara Di Ventura</p>
                    <p>BIOSS Centre and Institute of Biology II, University of Freiburg</p>
                    <p>
                        <button class="btn btn-abstract text-white border-white" type="button" data-bs-toggle="collapse" data-bs-target="#collapseAbstract3" aria-expanded="false" aria-controls="collapseAbstract3">
                            Read the abstract
                        </button>
                    </p>
                    <div class="collapse" id="collapseAbstract3">
                        <div class="card card-body text-white bg-black">
                            Cells respond to external stimuli and developmental cues mostly with changes in gene expression. Gene expression is often regulated by transcription factors (TFs), proteins that bind DNA in a sequence-specific manner at so-called response elements and recruit the general transcription machinery at the promoter to start transcription. TFs have been shown to display different dynamics, that is, the shape of the curve that describes the TF activity over time is not always the same, but depends on the type of stimulus the cell receives. Different dynamics have been, in turn, implicated in the activation of specific sets of target genes. In light of these findings, it is proposed that the same TF starts different gene expression programs purely depending on its dynamics and not its total abundance. This would imply that promoters be able to decode TF dynamics. In this lecture, I will present our work combining optogenetic experiments, automated image analysis and mathematical modeling to understand which elements on a mammalian promoter play a role in decoding TF dynamics. I will first describe LINuS, a light-inducible nuclear localization system developed by us, then show how, by imposing various TF dynamics with blue light and reading out the response from a library of synthetic promoters built with well-studied and defined elements, we find that sustained and pulsatile activation are distinguishable provided the coupling between TF binding and transcription pre-initiation complex formation is inefficient. Additionally, we show that the efficiency of translation initiation affects the ability of the promoter to sense TF dynamics. Using the knowledge acquired, we built a synthetic circuit that allows us to obtain two gene expression programs (proteins A and B both highly expressed versus protein A highly expressed and protein B only very weakly expressed) depending solely on TF dynamics. Taken together, these results help elucidate how gene expression is regulated in mammalian cells and open up the possibility to build synthetic circuits that respond better to determined TF dynamics.
                        </div>
                    </div>
                </div>
            </div>
            <div class="row program-item text-white bg-black py-3 px-0">
                <div class="col-md-2 pb-3">03.30 PM</div>
                <div class="col">
                    <p class="fs-5">Make them blink! - A chemistry approach to advanced fluorescence microscopy</p>
                    <p>by Dirk-Peter Herten</p>
                    <p>Institute of Cardiovascular Sciences and School of Chemistry, University of Birmingham</p>
                    <p>
                        <button class="btn btn-abstract text-white border-white" type="button" data-bs-toggle="collapse" data-bs-target="#collapseAbstract4" aria-expanded="false" aria-controls="collapseAbstract4">
                            Read the abstract
                        </button>
                    </p>
                    <div class="collapse" id="collapseAbstract4">
                        <div class="card card-body text-white bg-black">
                            Advanced fluorescence microscopies including but not limited to super-resolution techniques have seen tremendous progress over the past 15 years. While in the early days the developments were mainly driven by improvements of the optical microscopes and associated procedures for data acquisition, more recent developments with protein tags and click chemistry have opened new opportunities for improving and creating novel fluorescent probes significantly contributing to the advancements in fluorescence microscopy. In this context, we have explored the use of reversible and irreversible chemical reactions to control the emissive and none-emissive states of fluorescent probes. This not only enabled alternative approaches for multiplexing but also resulted in novel probes for single molecule localization microscopy (SMLM) and correlative microscopy approaches in living cells. Thorough studies of the involved photo physical process ultimately led to a new class of fluorescent probes that can be used in live-cell experiments without the need of further washing procedures.
                        </div>
                    </div>
                </div>
            </div>
            <div class="row program-item text-white bg-black py-3 px-0">
                <div class="col-md-2 pb-3">04.00 PM</div>
                <div class="col">
                    <p class="fs-5">Closing remarks</p>
                    <p>by Jürgen Wolfrum</p>
                    <p>Honorary Director of BioQuant</p>
                </div>
            </div>
            <div class="row program-item text-white bg-black py-3 px-0">
                <div class="col-md-2 pb-3">04.20 PM</div>
                <div class="col fs-5">End of meeting</div>
            </div>
            <div class="row program-item text-white bg-black py-3 px-0">
                <div class="col-md-2 pb-3">05.00 PM</div>
                <div class="col fs-5">Jubilee party</div>
            </div>

        </div>
    </section>

    <!-- Gallery section-->
    <section class="gallery text-white bg-black full-width">
        <div class="container">
            <div class="row">
                <div class="col-4 mb-4 mb-lg-0">
                    <img class="p-0 w-100 mb-4" alt="Computer cluster" src="/sites/default/files/15years/venue_1.jpg"/>
                    <img class="w-100 mb-4" alt="Staircase during nighttime" src="/sites/default/files/15years/venue_2.jpg"/>
                </div>
                <div class="col-4 mb-4 mb-lg-0">
                    <img class="w-100 mb-4" alt="Art installation" src="/sites/default/files/15years/venue_3.jpg"/>
                    <img class="w-100 mb-4" alt="Top floor" src="/sites/default/files/15years/venue_4.jpg"/>
                </div>
                <div class="col-4 mb-4 mb-lg-0">
                    <img class="w-100 mb-4" alt="Staircase during daytime" src="/sites/default/files/15years/venue_5.jpg"/>
                    <img class="w-100 mb-4" alt="Seminar room" src="/sites/default/files/15years/venue_6.jpg"/>
                </div>
            </div>
        </div>
    </section>

    <!--Registration section-->
    <section class="registration text-white bg-black full-width py-5">
        <div class="container" id="registration">
    
            <div class="fs-2 py-3">Registration</div>

            <p>Follow the link to register for the 15 Years of BioQuant symposium and jubilee party.</p>
            <p>Registration is open until Tuesday, June 14, 2022.</p>

            <a href="https://www.conferencecentral.org/signup/signup?t=ab1577db-ab97-4827-aba0-e80b7bd1b42f" class="btn btn-outline-secondary btn-register col-6 col-md-4 col-lg-3 mt-3 mb-4">
                Register here
            </a>

            <div class="row g-4 py-4">
                <div class="col-md-6">
                    <p class="fs-5">Do you want to join the events in person?</p>
                    <p>
                        Please note that the number of on-site participants is limited.<br>
                        We recommend that you register well in advance.
                    </p>
                </div>
                
                <div class="col-md-6">
                    <p class="fs-5">Do you plan to join the symposium online?</p>
                    <p>
                        The symposium is a hybrid event. All talks will be streamed online.<br>
                        When registering, please select the virtual participation option.
                    </p>
                </div>
            </div>

        </div>
    </section>
    
    <!--Venue section-->
    <section class="venue bg-grey full-width py-5">
        <div class="container" id="venue">
    
            <!--Section title-->
            <div class="fs-2 pb-3">How to Find Us</div>
            
            <div class="row g-4 py-4">
    
                <!--Text boxes-->
                <div class="col-sm-12 col-lg-7">
    
                    <!--Address-->
                    <div class="border px-0 mb-3">
                        <div class="bg-white text-center py-4">
                            <p class="text-green mt-2">Address</p>  
                            <p>BioQuant<br />Im Neuenheimer Feld 267<br />69120 Heidelberg</p>
                        </div>
                    </div>
    
                    <!--Directions-->
                    <div class="border px-0">
                        <div class="bg-white text-center py-4">
                            <p class="text-green mt-2">Public Transport</p>
                            <p class="mb-0">from Heidelberg Hbf</p>
                            <p>Tram No. 24 or 21 to Bunsengymnasium</p>
                            <p class="mb-0">from Bismarckplatz</p>
                            <p>Bus No. 31 or 29 to Bunsengymnasium</p>
                        </div>
                    </div>
                    
                </div>
    
                <!--Campus map-->
                <div class="col-sm-12 col-lg-5">
                    <div class="w-100"><img alt="Campus map" class="w-100 h-auto mb-0" src="/sites/default/files/2022-02/map.png" /></div>
                </div>
    
            </div>
    
        </div>
    </section>
    
    <!--Contact section-->
    <section class="contact py-5">
        <div class="container" id="contact">
    
            <div class="fs-2 py-3">Contact</div>
            
            <div class="row g-4 py-4 justify-content-between">
                <div class="col-sm-12 col-md-6">
                    <div class="border px-0">
                        <div class="d-block w-100 h-100 text-center text-black bg-grey py-4">
                            <p class="text-green mt-2">
                                Managing Director
                            </p>
                            <p class="fs-5 mt-2">
                                Dr. Katrin Hübner
                            </p>                       
                            <p>
                                <svg class="bi bi-envelope-fill" fill="currentColor" height="18" viewbox="0 0 18 18" width="18" xmlns="http://www.w3.org/2000/svg"> <path d="M.05 3.555A2 2 0 0 1 2 2h12a2 2 0 0 1 1.95 1.555L8 8.414.05 3.555ZM0 4.697v7.104l5.803-3.558L0 4.697ZM6.761 8.83l-6.57 4.027A2 2 0 0 0 2 14h12a2 2 0 0 0 1.808-1.144l-6.57-4.027L8 9.586l-1.239-.757Zm3.436-.586L16 11.801V4.697l-5.803 3.546Z"></path> </svg>
                                katrin.huebner @ bioquant.uni-heidelberg.de
                            </p>                      
                            <p>
                                <svg class="bi bi-telephone-fill" fill="currentColor" height="18" viewbox="0 0 18 18" width="18" xmlns="http://www.w3.org/2000/svg"> <path d="M1.885.511a1.745 1.745 0 0 1 2.61.163L6.29 2.98c.329.423.445.974.315 1.494l-.547 2.19a.678.678 0 0 0 .178.643l2.457 2.457a.678.678 0 0 0 .644.178l2.189-.547a1.745 1.745 0 0 1 1.494.315l2.306 1.794c.829.645.905 1.87.163 2.611l-1.034 1.034c-.74.74-1.846 1.065-2.877.702a18.634 18.634 0 0 1-7.01-4.42 18.634 18.634 0 0 1-4.42-7.009c-.362-1.03-.037-2.137.703-2.877L1.885.511z" fill-rule="evenodd"></path> </svg>
                                +49 (0)6221 54-51202
                            </p>
                            <p>
                                <svg class="bi bi-house-door-fill" fill="currentColor" height="18" viewbox="0 0 18 18" width="18" xmlns="http://www.w3.org/2000/svg"> <path d="M6.5 14.5v-3.505c0-.245.25-.495.5-.495h2c.25 0 .5.25.5.5v3.5a.5.5 0 0 0 .5.5h4a.5.5 0 0 0 .5-.5v-7a.5.5 0 0 0-.146-.354L13 5.793V2.5a.5.5 0 0 0-.5-.5h-1a.5.5 0 0 0-.5.5v1.293L8.354 1.146a.5.5 0 0 0-.708 0l-6 6A.5.5 0 0 0 1.5 7.5v7a.5.5 0 0 0 .5.5h4a.5.5 0 0 0 .5-.5z"></path> </svg>
                                INF 267 Room 741
                            </p>
                        </div>
                    </div>
                </div>
                <div class="col-sm-12 col-md-6">
                    <div class="border px-0">
                        <div class="d-block w-100 h-100 text-center text-black bg-grey py-4">
                            <p class="text-green mt-2">
                                Administration
                            </p>
                            <p class="fs-5 mt-2">
                                Michaela Schneider
                            </p>
                            <p><svg class="bi bi-envelope-fill" fill="currentColor" height="18" viewbox="0 0 18 18" width="18" xmlns="http://www.w3.org/2000/svg"> <path d="M.05 3.555A2 2 0 0 1 2 2h12a2 2 0 0 1 1.95 1.555L8 8.414.05 3.555ZM0 4.697v7.104l5.803-3.558L0 4.697ZM6.761 8.83l-6.57 4.027A2 2 0 0 0 2 14h12a2 2 0 0 0 1.808-1.144l-6.57-4.027L8 9.586l-1.239-.757Zm3.436-.586L16 11.801V4.697l-5.803 3.546Z"></path> </svg> 
                                geschaeftsstelle @ bioquant.uni-heidelberg.de
                            </p>
                            <p><svg class="bi bi-telephone-fill" fill="currentColor" height="18" viewbox="0 0 18 18" width="18" xmlns="http://www.w3.org/2000/svg"> <path d="M1.885.511a1.745 1.745 0 0 1 2.61.163L6.29 2.98c.329.423.445.974.315 1.494l-.547 2.19a.678.678 0 0 0 .178.643l2.457 2.457a.678.678 0 0 0 .644.178l2.189-.547a1.745 1.745 0 0 1 1.494.315l2.306 1.794c.829.645.905 1.87.163 2.611l-1.034 1.034c-.74.74-1.846 1.065-2.877.702a18.634 18.634 0 0 1-7.01-4.42 18.634 18.634 0 0 1-4.42-7.009c-.362-1.03-.037-2.137.703-2.877L1.885.511z" fill-rule="evenodd"></path> </svg> 
                                +49 (0)6221 54-51202
                            </p>                       
                            <p><svg class="bi bi-house-door-fill" fill="currentColor" height="18" viewbox="0 0 18 18" width="18" xmlns="http://www.w3.org/2000/svg"> <path d="M6.5 14.5v-3.505c0-.245.25-.495.5-.495h2c.25 0 .5.25.5.5v3.5a.5.5 0 0 0 .5.5h4a.5.5 0 0 0 .5-.5v-7a.5.5 0 0 0-.146-.354L13 5.793V2.5a.5.5 0 0 0-.5-.5h-1a.5.5 0 0 0-.5.5v1.293L8.354 1.146a.5.5 0 0 0-.708 0l-6 6A.5.5 0 0 0 1.5 7.5v7a.5.5 0 0 0 .5.5h4a.5.5 0 0 0 .5-.5z"></path> </svg> 
                                INF 267 Room 743
                            </p>
                        </div>
                    </div>
                </div>
            </div>
    
        </div>
    </section>    

</div>
